docs/website/docs/plus/getting-started/advanced_tutorial.md

```diff
@@ -1,304 +1,124 @@
-│   ├── dlt.yml           # the dlt+ project manifest
-│   └── ...               # more project files
+├── my_dlt_project/       # Your project module
+│   ├── __init__.py       # Package entry point
+│   ├── dlt.yml           # dlt+ project manifest
+│   └── ...               # Other project files
 ├── .gitignore
 └── pyproject.toml        # the main project manifest
 ```
 
-Your `dlt.yml` will look and work exactly the same as in the non-packaged version. The only difference is the module structure and the presence of the `pyproject.toml` file. The pyproject.toml has a special entry point setting that makes the dlt+ project discoverable by dlt+:
+Your `dlt.yml` works exactly the same as in non-packaged projects. 
+The key difference is the module structure and the presence of the `pyproject.toml` file. 
+The file includes a special entry point setting to let dlt+ discover your project:
 
 ```toml
 [project.entry-points.dlt_package]
 dlt-project = "my_project"
 ```
 
-You can still run the pipeline the same way as before with the cli commands from the root folder:
+You can still run the pipeline as usual with the cli commands from the root folder:
 
 ```sh
 dlt pipeline my_pipeline run
 ```
 
-If you look at the module `__init__.py` file that was created for you, you can see the full interface of your packaged project that the user of your package will have access to. It is very similar to the interface of the `current` interface of the flat project, there is an additional feature that selects the `access` profile by default. It is up to you to adapt this `__init__.py` file to your needs.
+If you open the `__init__.py` file inside your project module, you'll see the full interface that users of your package will interact with.
+This interface is very similar to the `current` interface used in flat (non-packaged) projects. The main difference is that it automatically uses the `access` profile by default.
+You can customize the `__init__.py` file to your project's needs.
+Learn more about the `current` interface and how it enables interaction with dlt+ projects [here](../features/projects.md#python_api_to_interact_with_dlt_project).
 
 ### Using the packaged project
 
-Using this project in its default state, let's simulate installing it into a new python project like a data-scientist in your organization might do, to demonstrate how this could be used in practice. For this example we use the uv package manager, but this is also possible with poetry or pip. Let's assume you have been developing your dlt+ packaged project at `/Volumes/my_drive/my_folder/pyproject.toml`. Navigate to a new directory and initialize your project:
+To demonstrate how your packaged project can be used, let's simulate a real-world scenario where a data scientist installs and runs your project in a separate Python environment.
+In this example, we'll use the **uv** package manager, but the same steps apply when using **poetry** or **pip**.
```

Comment:
Please, add link to the uv package manager: https://github.com/astral-sh/uv
and how to install it: https://github.com/astral-sh/uv?tab=readme-ov-file#installation